ci-operator/config/identitatem/idp-mgmt-operator/identitatem-idp-mgmt-operator-release-0.1.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+binary_build_commands: go build ./cmd/manager
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: idp-mgmt-operator
+promotion:
+  disabled: true
+  name: "0.1"
+  namespace: identitatem
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 80m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
```

Comment:
Don't need this stanza.

Reply:
Just copied from an open-cluster-management operator job...but will delete if not needed